### PR TITLE
Add new file browserslistrc

### LINKED ISSUE #109: Investigate Why Upgrading Browserslist Config Breaks in Downstream Projects
# Bug Report

## Description

Investigate why update browserslist-config-terra v1.2.0 to @cerner/browserslist-config-terra v^3.0.0 throws errors when running webpack.

See discussion on https://github.com/cerner/terra-application/pull/74

We need to figure this out to close #64 

## Steps to Reproduce

Follow steps outlined here to get started: https://github.com/cerner/terra-application/pull/74#pullrequestreview-474641279

## Expected Behavior

Terra-application's version of browserlist should not impact other project's webpack builds.

## Environment

* Node/npm Version: [e.g. Node 8/npm 5] lts/dubnium
* Webpack Version: ^4.43.0
* Operating System and version (desktop or mobile): Mac

## @ Mentions

@StephenEsser @avinashg1994

### PR DESCRIPTION
**Pull request by pranav300**
### Summary

This Pr
1. Adds new file browserslistrc which imports `@cerner/browserslist-config-terra`
2. Removes Browserslists from package.json.
 

Closes #109 #64 

### Deployment Link

https://terra-applic-.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: .browserslistrc [hunk 1]
```diff
@@ -0,0 +1 @@
+extends @cerner/browserslist-config-terra
```
Comment (emilyrohrbough) [inline .browserslistrc:1]: I'm pretty sure this will need to be ignored or we will run into the same issue that we are currently having.

Comment (ryanthemanuel) [reply .browserslistrc:1]: npm ignored right?

Comment (pranav300) [reply .browserslistrc:1]: Added here https://github.com/cerner/terra-application/pull/114/commits/042038071dfb6be41c14f2f291bebc18d16503c9.

**Review by ShettyAkarsh — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -11,6 +11,7 @@
 *.pid.lock
 *.rbc
 *.seed
+.browserslistrc
```
Comment (ryanthemanuel) [inline .gitignore]: This needs to not be gitignored but it does need to be in the npmignore file.

File: .browserslistrc (hunk 1 above)
Comment (ryanthemanuel) [reply .browserslistrc:1]: Looks like it's going to be ignored by this: https://github.com/cerner/terra-application/blob/main/package.json#L65-L71. Let's make sure this file doesn't get packed up when you run npm pack but I think we should be good without the npm ignore file.

File: .gitignore (hunk 2 above)
Comment (ryanthemanuel) [reply .gitignore]: Oops.  I think the npm ignore is covered by this: https://github.com/cerner/terra-application/blob/main/package.json#L65-L71 This still needs to be removed from the gitignore though

```suggestion
```

**Review by ryanthemanuel — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (pranav300) [reply .gitignore]: Updated here https://github.com/cerner/terra-application/pull/114/commits/438b2e1f36518f86e5b2a7083ae7d8893b692e7e.

**Review by mjhenkes — APPROVED** [review]

**Review by StephenEsser — APPROVED** [review]

**Pull request merged**